source/javascript/src/daily.js

```diff
@@ -27,6 +27,7 @@ cancelBtn.addEventListener('click', () => {
   text.value = ''
 })
 
+// TODO
```

Comment:
We should add an issue to the backlog to remove all `TODO` markers

Reply:
Ok will do